policyengine_us/parameters/gov/states/dc/tax/income/credits/ptc/fraction_elderly.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
       href: https://otr.cfo.dc.gov/sites/default/files/dc/sites/otr/publication/attachments/52926_D-40_12.21.21_Final_Rev011122.pdf#page=49
```

Comment:
could you add to the metadata:
uprating: gov.irs.uprating

This allows us to predict the values for future dates

Reply:
Could you do this for all yaml files that seem to increase every year with inflation

Reply:
note https://github.com/PolicyEngine/policyengine-core/issues/163 though

Reply:
yes, @shuyu7zhan could you add the uprating to the thresholds in the yaml files if they are updated with inflation 
See [CA](https://github.com/PolicyEngine/policyengine-us/pull/3770) as an example